app/views/secondary_content_links/index.html.erb

```diff
@@ -22,16 +22,8 @@
 %>
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-full">
-  <h2 class="govuk-heading-m">What is secondary content?</h2>
-    <div class="govuk-body">
-      <p class="govuk-body-s">Secondary content is content which is not linked to in the step by step, but has the step by step sidebar showing on it.</p>
-      <p class="govuk-body-s">We use it for content which:</p>
-      <ul class="govuk-list govuk-list--bullet">
-        <li class="govuk-body-s">helps you to complete a task in the journey, but is not the primary piece of content you need to visit to complete that task</li>
-        <li class="govuk-body-s">is optional and not useful enough to be included in the step by step</li>
-      </ul>
-    </div>
+  <div class="govuk-grid-column-full govuk-body-s">
+    <%= govspeak_to_html t("secondary_content_links.guidance") %>
```

Comment:
This should be inside a govspeak component by the way to render the markdown correctly.

Reply:
I don't think this will be needed now we're not using Govspeak.  All the markdown should have been parsed into valid HTML.

Reply:
We'll still need it to add the styles to the HTML as our naked HTML is pretty much styleless with govuk-frontend.

Reply:
The relevant classes are already there, since we had plain HTML in the places where this text was previously appearing.

Reply:
After chatting in person earlier, I've added a `render_markdown` method which wraps the HTML in a govspeak component.

I've also reduced the number of fields in which we allow markdown, since input labels shouldn't need to contain markdown.